src/WorldIDIdentityManagerImplV1.sol

```diff
@@ -168,6 +168,8 @@ contract WorldIDIdentityManagerImplV1 is WorldIDImpl, IWorldID {
     ///
     /// @param index The index in the array of identity commitments where the invalid commitment was
     ///        found.
+    /// @dev This error is no longer in use as we now verify the commitments off-chain within the circuit
+    /// no need to check for reduced elements or invalid commitments.
     error InvalidCommitment(uint256 index);
```

Comment:
Can we remove this if it is no longer needed?

Reply:
wouldn't it change the storage structure?

Reply:
This would not change the storage structure. 

However after thinking this through a bit more, if we removed these errors (as well as unused enums and events) it would make it challenging to decode logs/errors from previous transactions. 

For example, if for any reason someone wanted to decode logs from the contract before this upgrade, they would need the previous contract ABI. 

There seems to be little benefit for removing as the optimizer should remove unused code and it will not have an affect on code size or gas consumption.  I think we should leave this code for backwards compatibility with the previous contract ABI.